packages/rich-text/src/get-active-formats.js

```diff
@@ -1,9 +1,13 @@
 /**
  * Gets the all format objects at the start of the selection.
  *
- * @param {Object} value                Value to inspect.
- * @param {Array}  EMPTY_ACTIVE_FORMATS Array to return if there are no active
- *                                      formats.
+ * @param {Object}        value                Value to inspect.
+ * @param {Array<Object>} value.formats        Formats object data values.
```

Comment:
There are no objects at the positions, but rather arrays. Each position can contain multiple formats. The position can also be empty, but I'm not sure how this would be documented.

For example, `formats` can be: `[ , , [ { type: 'core/bold' } ] ]`.

Reply:
```suggestion
 * @param {Array<Array>} value.formats        Formats object data values.
```

Reply:
Thanks @ellatrix ,  I have made changes as per your suggestion.